src/Containers.jsx

```diff
@@ -226,12 +225,12 @@ class Containers extends React.Component {
         }
 
         return {
-            expandedContent: <ScrollableAnchor id={container.Id} key={container.Id}>
```

Comment:
Wait, I misunderstood what this does -- it was introduced in commit b5f8df880253e2d. I'll check what that actually renders to, and whether it's enough to just replace it with a simple HTML tag.

Reply:
Lol -- this functionality is broken in master. Once I click on the "used by" container, the page crashes with `TypeError: props.showAll is not a function`.

Reply:
Fixed in #705, but the actual scrolling is still broken.